ci-operator/config/openshift/osde2e/openshift-osde2e-main.yaml

```diff
@@ -403,6 +403,23 @@ tests:
     name: osde2e-common
   - mount_path: /usr/local/osde2e-credentials
     name: osde2e-credentials
+- as: rosa-stage-complete-e2e
+  commands: "export REPORT_DIR=\"$ARTIFACT_DIR\"\nexport CONFIGS=\"rosa,stage,complete-e2e\"\nexport
+    CHANNEL=\"stable\"\nexport SECRET_LOCATIONS=\"/usr/local/osde2e-common,/usr/local/osde2e-credentials,/usr/local/osde2e-rosa-stage\"\nexport
+    SUITE_TIMEOUT=\"900\"\nexport POLLING_TIMEOUT=\"7200\"\nexport ROSA_ENV=\"stage\"\nexport
+    ROSA_STS=\"true\"  \nexport ROSA_AWS_REGION=\"random\"\n\n/osde2e test --secret-locations
+    ${SECRET_LOCATIONS} --configs ${CONFIGS}\n"
```

Comment:
```suggestion
  commands: |
    export REPORT_DIR="$ARTIFACT_DIR"
    export CONFIGS="rosa,sts,stage,complete-e2e"
    export CHANNEL="stable"
    export ROSA_AWS_REGION="random"
    export SECRET_LOCATIONS="/usr/local/osde2e-common,/usr/local/osde2e-credentials,/usr/local/osde2e-rosa-stage"

    /osde2e test --secret-locations ${SECRET_LOCATIONS} --configs ${CONFIGS}
```

Reply:
@ritmun That is interesting that it is forcing it. I just pulled the changes locally, and ran both `make jobs` and `make ci-operator-config` and it allowed me to keep the formatting as I suggested. The strange thing is, if its enforcing this formatting, I would have expected it to perform the same operation for all the other jobs within the file.

Maybe we try the following:

1. Rebase this PR with latest changes from upstream repository
2. Run `make jobs` and `make ci-operat-config` with the new formatting for `commands`
3. Check in the changes
4. Lets see what the PR checks present to us

I would have expected to reproduce the same behavior locally, are you able to reproduce the error the PR check was failing on?

Reply:
Just as a reference here are the images that the make targets pulled when running locally on my laptop:

```
$ podman images
REPOSITORY                                                 TAG          IMAGE ID      CREATED        SIZE
registry.ci.openshift.org/ci/determinize-ci-operator       latest       7b6e9f96f8ef  14 hours ago   399 MB
registry.ci.openshift.org/ci/sanitize-prow-jobs            latest       b1760df9023c  14 hours ago   306 MB
registry.ci.openshift.org/ci/ci-operator-checkconfig       latest       34c9462d67d6  14 hours ago   401 MB
registry.ci.openshift.org/ci/ci-operator-prowgen           latest       118813d1aec0  14 hours ago   400 MB
```